config.json.sample

```diff
@@ -9,8 +9,8 @@
   "resolvers": ["8.8.8.8"],
   "timeout": 5,
   "listener": "0.0.0.0",
-  "SOAMname": "root.ns1.mesos",
-  "SOARname": "ns1.mesos",
+  "SOAMname": "ns1.mesos",
```

Comment:
should we specify a sample name that's fully qualified? (ends in a dot)

e.g. `ns1.mesos.`